.travis.yml

```diff
@@ -18,8 +18,8 @@ sudo: required
 addons:
   postgresql: "9.3"
 before_install:
-  - wget "https://github.com/Masterminds/glide/releases/download/0.8.1/glide-0.8.1-linux-amd64.tar.gz"
-  - sudo tar -vxz -C /usr/local/bin --strip=1 -f glide-0.8.1-linux-amd64.tar.gz
+  - wget "https://github.com/Masterminds/glide/releases/download/0.8.3/glide-0.8.3-linux-amd64.tar.gz"
+  - sudo tar -vxz -C /usr/local/bin --strip=1 -f glide-0.8.3-linux-amd64.tar.gz
```

Comment:
I understand, but while this fixes it for you, it might break it for someone else.  This seems like impetus to containerize the build / test of the go client using deis/docker-go-dev.  That would insulate all contributors from these sort of environmental differences.